exporter/opentelemetry-exporter-prometheus/src/opentelemetry/exporter/prometheus/__init__.py

```diff
@@ -78,14 +79,13 @@
 def _convert_buckets(metric: Metric) -> Sequence[Tuple[str, int]]:
     buckets = []
     total_count = 0
-    for index, value in enumerate(metric.point.bucket_counts):
-        total_count += value
-        buckets.append(
-            (
-                f"{metric.point.explicit_bounds[index]}",
-                total_count,
-            )
-        )
+    for upper_bound, count in zip(
+        chain(metric.point.explicit_bounds, ["+Inf"]),
+        metric.point.bucket_counts,
```

Comment:
won't this fail to produce an `+Inf` bucket if `bucket_counts` is less than the length of `explicit_bounds`? Is that a case we need to test?

Reply:
`bucket_counts` must have length of `1 + len(explicit_bounds)`. If it doesn't there is an issue with the SDK, so not worth testing here I don't think.